package.json

```diff
@@ -83,9 +81,9 @@
     "prettier": "^2.8.1",
     "qunit": "^2.19.3",
     "qunit-dom": "^2.0.0",
-    "release-it": "^14.2.1",
-    "release-it-lerna-changelog": "^3.1.0",
-    "rollup": "^2.57.0",
+    "release-it": "^15.5.1",
+    "release-it-lerna-changelog": "^5.0.0",
+    "rollup": "^3.7.5",
```

Comment:
I tested `rollup` by running `yarn start` at the root directory. I was able to visit http://localhost:4200 and see the demo app render the `<WelcomePage>` component.